packages/gestalt/src/PopoverEducational.js

```diff
@@ -162,7 +162,7 @@ export default function PopoverEducational({
 
   return (
     <Box zIndex={zIndex}>
-      <Popover
+      <InternalPopover
```

Comment:
Line 174 is redundant as we set a default.  Workrth try cleaning up

Reply:
DONE